lib/Marlin/Marlin/src/module/temperature.cpp

```diff
@@ -3845,7 +3845,8 @@ void Temperature::isr() {
     #if HAS_HEATED_CHAMBER
```

Comment:
"Make sure to add chamber temp cap + report only if in chamber capabil…"
should not be a separate commit - the functionality should be implemented correctly from the go